README.md

```diff
@@ -1,33 +1,21 @@
-# Trilium Notes
-
-## Trilium is in maintenance mode - see details in https://github.com/zadam/trilium/issues/4620
-
-Preliminary disccusions on the successor organization are taking place in [Trilium Next discussions](https://github.com/orgs/TriliumNext/discussions). 
+# TriliumNext Notes
 
 [English](https://github.com/zadam/trilium/blob/master/README.md) | [Chinese](https://github.com/zadam/trilium/blob/master/README-ZH_CN.md) | [Russian](https://github.com/zadam/trilium/blob/master/README.ru.md) | [Japanese](https://github.com/zadam/trilium/blob/master/README.ja.md) | [Italian](https://github.com/zadam/trilium/blob/master/README.it.md)
 
+TriliumNext Notes is a hierarchical note taking application with focus on building large personal knowledge bases. 
 
-Trilium Notes is a hierarchical note taking application with focus on building large personal knowledge bases. 
-
-## Discuss with us
-
-Feel free to join our discussions.
-
-- [XMPP](https://joinjabber.org/): [xmpp:discuss@trilium.thisgreat.party?join](xmpp:discuss@trilium.thisgreat.party?join) ([web link](https://anonymous.cheogram.com/discuss@trilium.thisgreat.party))
-- [Matrix](https://matrix.org/try-matrix/): #trilium:matrix.org ([web link](https://app.element.io/#/room/#trilium:matrix.org))
-
-The two rooms are mirrored, so you can use the protocol of your choice, from the client you prefer, on pretty much any platform under the sun!
+<a href="https://github.com/zadam/trilium/wiki/Screenshot-tour"><img src="https://raw.githubusercontent.com/wiki/zadam/trilium/images/screenshot.png" alt="Trilium Screenshot" width="1000"></a>
```

Comment:
Might as well include the screenshots and tour in this repo and not link to zadams.